bin/NativeTests/Scripts/splay.js

```diff
@@ -2,7 +2,7 @@
 // base.js
 ////////////////////////////////////////////////////////////////////////////////
 
-// Copyright 2008 the V8 project authors. All rights reserved.
+// Copyright 2013 the V8 project authors. All rights reserved.
```

Comment:
Not sure if this copyright year should be updated, if in fact the version of the file you have here is from 2008. Maybe hold off on this until you can fix the test w.r.t. the version we already have in the repo?